.github/workflows/doc-build.yml

```diff
@@ -82,7 +87,38 @@ jobs:
                 set -o pipefail;
                 make html | tee log.txt;
         -   name: Upload documentation
-            uses: actions/upload-artifact@v3
+            uses: actions/upload-artifact@v4
             with:
-                name: doc
+                name: DocHTML
                 path: doc/_build/html
+                
+    # Deploy the documentation to GitHub Pages.
+    deploy_docs:
+        if: github.repository == 'parietal-INRIA/fmralign'
+        needs: build_docs
+        runs-on: ubuntu-latest
+        steps:
+        -   uses: actions/checkout@v3
+
+        -   name: Download artifacts
+            uses: actions/download-artifact@v4
+            with:
+                name: DocHTML
+                path: doc/_build/html/
```

Comment:
```suggestion
                name: DocHTML
                path: doc/_build/html/
                fetch-depth: 0
```

Reply:
Looking at the examples here(https://github.com/ad-m/github-push-action), I think we might be missing `fetch-depth: 0`

Reply:
Good point I'm adding it 👍